### DISCUSSION TITLE
Backups?

### DISCUSSION THREAD

**Discussion by jarrah31** [Koenkk/zigbee2mqtt#7734] [General]
If my Pi SD card were to die, or the zigbee coordinator module failed, what is the proper way to backup and restore my config so that I don't need to repair everything please?

Is it as simple as making a copy of the data folder and restoring that after a fresh install?

1.90.x says `The commissioning and backup procedure has heavily been refactored.` - does this change the backup/restore procedure in any way?

Would it be possible to integrate saving a backup zip file via the GUI as an easier way to copy the data off device please?

**Comment by Koenkk** [comment]
> Is it as simple as making a copy of the data folder and restoring that after a fresh install?

Yes

> does this change the backup/restore procedure in any way?

No, only the `coordinator_backup.json` format has been changed, however z2m 1.19.0 supports both the old and new format

> Would it be possible to integrate saving a backup zip file via the GUI as an easier way to copy the data off device please?

Currently not, maybe in the future.
